stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/messages/MessageText.kt

```diff
@@ -122,6 +123,66 @@ public fun MessageText(
     }
 }
 
+/**
+ * Default text element for quoted messages, with extra styling and padding for the chat bubble.
+ *
+ * @param message Message to show.
+ * @param modifier Modifier for styling.
+ */
+@Composable
+public fun QuotedMessageText(
+    message: Message,
+    modifier: Modifier = Modifier,
+    maxQuoteLength: Int = DefaultQuoteMaxLength,
+) {
+    val context = LocalContext.current
+
+    val quotedMessage = when {
+        message.text.count() > DefaultQuoteMaxLength -> {
+            val text = context.getString(
+                R.string.stream_compose_quoted_message_ellipsize,
+                message.text.take(maxQuoteLength)
+            )
+            message.copy(text = text)
+        }
+
+        message.text.isBlank() -> {
+            val attachment = message.attachments.first()
+            val text = when {
+                attachment.type == ModelType.attach_image -> {
+                    context.getString(R.string.stream_compose_quoted_message_image_tag)
+                }
+                attachment.type == ModelType.attach_giphy -> {
+                    context.getString(R.string.stream_compose_quoted_message_giphy_tag)
+                }
+                attachment.isFile() -> {
+                    attachment.name ?: context.getString(R.string.stream_compose_quoted_message_file_tag)
+                }
+                else -> message.text
+            }
+            message.copy(text = text)
+        }
+
+        else -> message
+    }
+
+    val styledText = buildAnnotatedMessageText(quotedMessage)
+
+    val horizontalPadding = 8.dp
+    val verticalPadding = 5.dp
+
+    Text(
+        modifier = modifier.padding(
+            top = verticalPadding,
```

Comment:
there is a `padding()` variant with horizontal and vertical parameters and we can inline them rather than pulling them out as as a value